core/src/main/scala/fs2/util/syntax.scala

```diff
@@ -0,0 +1,26 @@
+package fs2.util
+
+/** Provides infix syntax for the typeclasses in the util package. */
```

Comment:
We could make this private if we really wanted to make it for internal use only but I'm fine with keeping it public. It requires explicit import.